.prettierrc.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
-  arrowParens: 'avoid',
+  arrowParens: 'always',
   bracketSameLine: false,
   bracketSpacing: true,
   singleQuote: true,
   trailingComma: 'all',
-  printWidth: 80,
+  printWidth: 120,
```

Comment:
I reverted this to 80 for now. My thoughts on this are evolving but I'm currently thinking that since this is something that should run on save, it shouldn't be too difficult to get used to the different ones between each project. You shouldn't have to think about it, it should just work. What do you think?